source/index.html.haml

```diff
@@ -1,19 +1,19 @@
 ---
-title: RubyConf - Colombia
+title: RubyConf Colombia
 ---
 
-= partial "partials/registration"
+/ = partial "partials/registration"
 
-/ = partial "partials/venue"
+= partial "partials/tickets"
 
 = partial "partials/speakers"
 
-/ = partial "partials/tickets"
-
 / = partial "partials/schedule"
 
-= partial "partials/sponsors"
+/ = partial "partials/venue"
```

Comment:
Not yet, we still need to add the map. If want to add that, we can activate it :smile: